main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"log"
+	"lolidelgado/github-users/controller"
+	"lolidelgado/github-users/delivery"
+	"lolidelgado/github-users/repository"
+	"lolidelgado/github-users/usecase"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/unrolled/render"
```

Comment:
A good practice in Go is to separate all your imports into three main blocks:
- 1st - Standard library packages
- 2nd - Internal (project) packages
- 3rd - Third party packages

You can swap 2nd and 3rd blocks, and also each block should contain all of its elements in alphabetical order.

```suggestion
	"log"
	"net/http"
	
	"lolidelgado/github-users/controller"
	"lolidelgado/github-users/delivery"
	"lolidelgado/github-users/repository"
	"lolidelgado/github-users/usecase"

	"github.com/gorilla/mux"
	"github.com/unrolled/render"
```

Please fix this and all other occurrences of this package blocking issue.

Reply:
Ordering and separating the packages following that good practice, in fact, I think when saving the files the imports were automatically ordered, maybe I missed something the first time I created them. Thanks!